zipline/data/data_portal.py

```diff
@@ -637,7 +638,51 @@ def _check_extra_sources(self, asset, column, dt):
 
                 raise KeyError
 
-    def get_spot_value(self, asset, field, dt, data_frequency):
+    def _get_overnight_adjustments(self, assets, field, dt):
+
+        if not isinstance(assets, (list, tuple)):
+            assets = [assets]
+
+        # dt at bts is forced back to 4 pm previous day for the purpose of
```

Comment:
Is this the previous day, or the previous **trading day**?  If the latter, then this is going to be broken on mondays/holidays/etc.